openfisca_aotearoa/variables/rates_rebates.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and benefit system
+from openfisca_aotearoa.entities import Propertee, Person
+from numpy import clip
+
+
+# This is a 'temporary' variable used specifically for the rates rebates
+# Due to the structure not yet being clear, this may need to be moved, renamed.
+class dependants(Variable):
+    value_type = int
+    entity = Person
+    definition_period = YEAR
+    label = u"Number of Persons classified as dependant on the entity Person for the purposes of rates rebates"
+
+
+class rates(Variable):
+    value_type = float
+    entity = Propertee
+    definition_period = YEAR
+
+
+class rates_rebate(Variable):
+    value_type = float
+    entity = Propertee
+    definition_period = YEAR
+    label = "Yearly rebate applied to housing rates. Defined on a calendar year, not a fiscal year."
+    reference = ""
```

Comment:
If this is also coming from the spreadsheet it would be useful to mark it here 🙂

Reply:
Done. Included comment with link to actual legislation but left reference as spreadsheet as that was what was worked from.